src/stories/nve-tooltip/NveTooltipDoc.mdx

```diff
@@ -0,0 +1,48 @@
+import './../css/stories.css';
+
+import {
+  Canvas,
+  Story,
+  Meta,
+  Title,
+  Subtitle,
+  Description,
+  Source,
+  Primary,
+  Controls,
+  Stories,
+} from '@storybook/blocks';
+
+import * as NveTooltipStories from './NveTooltip.stories';
+
+<Meta isTemplate />
+
+<Title />
+<Description />
+<br />
+<br />
+
+## Standard
+
+<div class="container">
+  <span class="elem">
+    <Canvas of={NveTooltipStories.Hjelpetekst} />
+  </span>
+</div>
+
+## Med HTML
+
+Bruk slot "content" for å bruke html i tooltip.
+
+<div class="container">
+  <span class="elem">
+    <Canvas of={NveTooltipStories.Html} />
+  </span>
+</div>
+
```

Comment:
Jeg tror vi kommer til å bruke tooltip mer i tilknytning til label. Nå blir dette sikkert vist i nve-label, men kanskje det hadde vært nyttig å se det her og i tilfelle folk ser her i stedet for i nve-label når de lurer på dette med toooltip?